config/application.rb

```diff
@@ -37,6 +37,8 @@ class Application < Rails::Application
       g.orm :active_record, primary_key_type: :uuid
     end
 
+    config.eager_load_paths += %W[#{config.root}/lib]
+
```

Comment:
I have a question about this line of code. Is it here where you are telling Rails to eager load from the file you created on `lib` that contains your `FinancialPeriod` module?

Reply:
Yep. We using `+=` so we are adding a path the one already configured.